projects/kit/src/lib/utils/time/parse-time-string.ts

```diff
@@ -14,11 +14,14 @@ export function parseTimeString(
     timeString: string,
     timeMode: MaskitoTimeMode,
 ): Partial<MaskitoTimeSegments> {
```

Comment:
Add unit tests for this utility too

https://github.com/taiga-family/maskito/blob/41ab86d723afa6d5b85eee83861c79dd471c8ae7/projects/kit/src/lib/utils/time/tests/parse-time-string.spec.ts#L3-L5